.github/workflows/copilot.yml

```diff
@@ -0,0 +1,59 @@
+name: Copilot Workflows
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+        environment:
+          description:  Which AWS Account to use
+          type: choice
+          required: true
+          options:
+          - test
+        # Shared workflow consideration
+        # application:
+        #   description:  Application Name
+        #   type: string/choice
+        # - notification
+        #   required: true
+        init:
+          description: Initialise the application?
+          type: boolean
+          default: false
+        service:
+          description: Service Name
+          type: string
+          required: true
+          default: 'funding-service-design-frontend'
+        port:
+          description: Access port
+          type: string
+          default: '80'
+        type:
+          description: Type of service to deploy
+          type: choice
+          options:
+          - 'Backend Service'
+          - 'Load Balanced Web Service'
+          - 'Request-Driven Web Service'
+          - 'Scheduled Job'
+          - 'Worker Service'
+          default: 'Backend Service'
```

Comment:
I guess since it's a "dummy" workflow these are just the placeholder inputs?

Reply:
Correct